README.md

```diff
@@ -1,77 +1,94 @@
-# Role Name
-![template](https://github.com/linux-system-roles/template/workflows/tox/badge.svg)
+# PostgreSQL system role
+![CI Testing](https://github.com/linux-system-roles/template/workflows/tox/badge.svg)
 
-A template for an ansible role which configures some GNU/Linux subsystem or
-service. A brief description of the role goes here.
+This role installs, configures, and starts PostgreSQL Server.
 
-## Requirements
-
-Any pre-requisites that may not be covered by Ansible itself or the role should
-be mentioned here. For instance, if the role uses the EC2 module, it may be a
-good idea to mention in this section that the `boto` package is required.
+The role also optimizes the datadase server settings to improve performance.
 
+The role currently works with PostgreSQL server 10 12 and 13.
 ## Role Variables
-
-A description of all input variables (i.e. variables that are defined in
-`defaults/main.yml`) for the role should go here as these form an API of the
-role.
-
-Variables that are not intended as input, like variables defined in
-`vars/main.yml`, variables that are read from other roles and/or the global
-scope (ie. hostvars, group vars, etc.) can be also mentioned here but keep in
-mind that as these are probably not part of the role API they may change during
-the lifetime.
-
-Example of setting the variables:
-
+### postgresql_verison
+Allow set up version of Postgresql server. This role supports Postgresql 10 12 and 13
 ```yaml
-template_foo: "oof"
-template_bar: "baz"
+postgresql_version: "13"
+```
+### postgresql_password
+Set up default password for database super user `postgres`
+```yaml
+postgresql_password: "mysecretpassword"
```

Comment:
Recommend using Ansible Vault to encrypt the value. See https://docs.ansible.com/ansible/latest/user_guide/vault.html